packages/twenty-server/src/modules/messaging/message-import-manager/utils/__tests__/is-sync-stale.util.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { MESSAGING_IMPORT_ONGOING_SYNC_TIMEOUT } from 'src/modules/messaging/message-import-manager/constants/messaging-import-ongoing-sync-timeout.constant';
+import { isSyncStale } from 'src/modules/messaging/message-import-manager/utils/is-sync-stale.util';
+
+describe('isSyncStale', () => {
+  it('should return true if sync is stale', () => {
+    const syncStageStartedAt = new Date(
+      Date.now() - MESSAGING_IMPORT_ONGOING_SYNC_TIMEOUT - 1,
```

Comment:
this might be flaky. We should use: https://stackoverflow.com/questions/29719631/how-do-i-set-a-mock-date-in-jest